.github/workflows/zowe-versions_updateVersions_createPR.yml

```diff
@@ -0,0 +1,85 @@
+name: Update Versions and Create PR (zowe-versions)
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release Version (x.x.x)'
+        required: true
+      zowe_version:
+        description: "Zowe Version Tag"
+        default: zowe-v2-lts
+        required: true
+        type: choice
+        options:
+        - zowe-v1-lts
+        - zowe-v2-lts
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+
+      - name: Create Branch
+        run: |
+          version_name="${{ github.event.inputs.version }}"
+          branch_name="update-versions-${version_name}"
+          git checkout -b "$branch_name"
+
+      - name: Get and Change Component Versions
+        id: get_versions
+        run: |
+          zowe_version="${{ github.event.inputs.zowe_version }}"
+          components=$(yq '.packages | with_entries(select(.value."$zowe_version")) | keys | .[]' zowe-versions.yaml)
+          version="${{ github.event.inputs.version }}"
+          if [[ "$zowe_version" == "zowe-v1-lts" ]]; then
+            components+=("secure-credential-store-for-zowe-cli")
+          fi
+          # echoing to txt file to later add this file as the PR description
+          echo "This PR updates the following package versions in zowe-versions.yaml for \`$zowe_version\`" >> description.txt
+          echo "| Updated Package | Old Version | New Version |" >> description.txt
+          echo "| --- | --- | --- |" >> description.txt
+          for component in "${components[@]}"; do
+            current_version=$(grep -A 2 "^  $component:" zowe-versions.yaml | grep "$zowe_version" | awk '{print $2}')
+            new_version=$(npm view @zowe/$component "dist-tags.$zowe_version")
+            if [[ "$new_version" != "$current_version" ]]; then
+              echo "|$component | $current_version | $new_version|" >> description.txt
+              # Replaces the version number. If it doesn't match on the first line after the component, it goes to the next line and checks again.
+              sed -i -E "/^  $component:/ { n; s/($zowe_version: ).*/\1$new_version/; t; n; s/($zowe_version: ).*/\1$new_version/ }" zowe-versions.yaml
+            fi
+          done
+
+          # Update zowe-versions.yaml tags section
+          sed -i -E "/$zowe_version:/ { n; s/(version: ).*/\1$version/ }" zowe-versions.yaml

```

Comment:
In zowe-versions.yaml, on the line after `version: X.Y.Z` we have `rc: N` to define the Release Candidate number.

Should the RC number be reset to 1 in this workflow? Otherwise if the last RC for the previous release was &gt; 1, we'll have to manually reset it.